codecov.yml

```diff
@@ -0,0 +1,32 @@
+codecov:
+  branch: develop    # the branch to show by default
+
+  # The help text for bot says:
+  # "the username that will consume any oauth requests
+  # must have previously logged into Codecov"
+  # In GitHub - BigchainDB organization settings - Third-party access,
+  # it says, for Codecov: "approval requested by r-marques"
+  bot: r-marques
```

Comment:
I don't know if @wrigley has previously logged into Codecov, and I have no way of checking.